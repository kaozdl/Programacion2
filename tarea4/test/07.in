# cola de prioridad
cola_prioridad 2 .
cola_prioridad 2 (5,a) .
cola_prioridad 5 (5,b) (5,c) .
cola_prioridad 5 (4,d) .
cola_prioridad 4 (7,a) (3,g) .
cola_prioridad 3 (15,a) (3,g) (5,d) .
cola_prioridad 5 (12,y) (8,g) (18,a) (20,g) (45,y) (12,g) .
cola_prioridad 12 .
cola_prioridad 20 (2,a) (28,b) (3,c) (11,d) (1,y) (1,x) .
cola_prioridad 1 (10,z) (30,y) (17,x) (99,no) (98,no) (1,si) .
cola_prioridad 1 .
cola_prioridad 1 .
cola_prioridad 99 .
cola_prioridad 45 .
cola_prioridad 30 .
cola_prioridad 2 .
cola_prioridad 2 .
cola_prioridad 2 .
cola_prioridad 2 .
cola_prioridad 2 .
cola_prioridad 2 .
cola_prioridad 2 .
cola_prioridad 2 .
cola_prioridad 2 .
cola_prioridad 2 .
cola_prioridad 2 .
cola_prioridad 2 .
Fin
