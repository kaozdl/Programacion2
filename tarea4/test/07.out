1>
2>
3>
(5,a)
4>
(5,b)
5>(5,c)
(4,d)
6>
(3,g)
7>
(3,g)
8>(5,c)(5,d)
(5,c)
9>(12,y)(12,g)
(5,d)
10>(20,g)
(1,y)
11>(1,x)
(1,x)
12>(1,si)
(1,si)
13>
(2,a)
14>(99,no)
(3,c)
15>(45,y)
(7,a)
16>(30,y)
(8,g)
17>
(10,z)
18>
(11,d)
19>
(12,y)
20>
(12,g)
21>
(15,a)
22>
(17,x)
23>
(18,a)
24>
(20,g)
25>
(28,b)
26>
(30,y)
27>
(45,y)
28>
(98,no)
29>Fin.
